packages/truffle-environment/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "truffle-environment",
+  "version": "0.1.0",
+  "description": "This package contains the code needed to determine environment-specific data",
+  "main": "index.js",
+  "author": "Faina Shalts",
+  "license": "MIT",
```

Comment:
Oh, can we get a LICENSE file added to the pkg?

Reply:
Yep!